ui/raidboss/data/timelines/e1s.txt

```diff
@@ -11,88 +11,73 @@ hideall "--sync--"
+576.1 "Spear Of Paradise" sync /:Eden Prime:3D88:/
+577.7 "Heavensunder" sync /:Eden Prime:3D89:/
+582.5 "--sync--" sync /:Eden Prime:4683:/
+589.1 "Dimensional Shift" sync /:Eden Prime:3D7F:/
+599.5 "Pure Beam!" sync /:Eden Prime:3D82:/
+611.2 "--corner--" sync /:Eden Prime:4683:/
+619.9 "Pure Light" sync /:Eden Prime:3D8A:/ duration 11.7
+622.8 "--center--" sync /:Eden Prime:4683:/
 
-629.7 "Dimensional Shift" sync /:Eden Prime:3D7F:/
-642.4 "Eden's Gravity" sync /:Eden Prime:3D70:/
-654.0 "Eden's Flare" sync /:Eden Prime:3D73:/
-669.6 "Vice And Virtue (T)" sync /:Eden Prime:44EE:/
-677.1 "Spear Of Paradise" sync /:Eden Prime:3D88:/
-678.6 "Heavensunder" sync /:Eden Prime:3D89:/
-688.4 "Vice And Virtue (DPS)" sync /:Eden Prime:44EF:/
-690.9 "--sync--" sync /:Eden Prime:4683:/
-701.0 "Delta Attack (Cross)" sync /:Eden Prime:44F4:/
-714.6 "Vice And Virtue (H)" sync /:Eden Prime:44F0:/
-720.2 "--sync--" sync /:Eden Prime:4683:/
-726.8 "Dimensional Shift" sync /:Eden Prime:3D7F:/
-737.3 "Pure Beam!" sync /:Eden Prime:3D82:/ duration 11.7
-748.9 "--corner--" sync /:Eden Prime:4683:/
-757.7 "Pure Light" sync /:Eden Prime:3D8A:/
-760.5 "--center--" sync /:Eden Prime:4683:/
-767.1 "Dimensional Shift" sync /:Eden Prime:3D7F:/
-779.8 "Eden's Gravity" sync /:Eden Prime:3D70:/
-791.4 "Eden's Flare" sync /:Eden Prime:3D73:/
-
-802.0 "Paradise Regained" sync /:Eden Prime:44FC:/ window 200,200 jump 518.3
-812.5 "Vice And Virtue! (T)"
-826.1 "Vice And Virtue! (D)"
-828.6 "--sync--"
-838.7 "Delta Attack! (Donut)"
-852.4 "Vice And Virtue! (H)"
-860.0 "Spear Of Paradise"
-861.5 "Heavensunder"
-866.3 "--jump--"
-872.9 "Dimensional Shift"
-883.4 "Pure Beam!"
-895.2 "--corner--"
-904.0 "Pure Light"
-906.8 "--center--"
-913.4 "Dimensional Shift"
-926.1 "Eden's Gravity"
-937.7 "Eden's Flare"
-
-### Enrage??
\ No newline at end of file
+629.3 "Dimensional Shift" sync /:Eden Prime:3D7F:/
+642.1 "Eden's Gravity" sync /:Eden Prime:3D70:/
+653.6 "Eden's Flare" sync /:Eden Prime:3D73:/
+669.2 "Vice And Virtue (T)" sync /:Eden Prime:44EE:/
+676.8 "Spear Of Paradise" sync /:Eden Prime:3D88:/
+678.3 "Heavensunder" sync /:Eden Prime:3D89:/
+688.0 "Vice And Virtue (DPS)" sync /:Eden Prime:44EF:/
+690.5 "--sync--" sync /:Eden Prime:4683:/
+700.6 "Delta Attack (Cross)" sync /:Eden Prime:44F4:/
+714.1 "Vice And Virtue (H)" sync /:Eden Prime:44F0:/
+719.8 "--sync--" sync /:Eden Prime:4683:/
+726.4 "Dimensional Shift" sync /:Eden Prime:3D7F:/
+736.9 "Pure Beam!" sync /:Eden Prime:3D82:/ duration 11.7
+748.3 "--corner--" sync /:Eden Prime:4683:/
+757.1 "Pure Light" sync /:Eden Prime:3D8A:/
+760.0 "--center--" sync /:Eden Prime:4683:/
+766.5 "Dimensional Shift" sync /:Eden Prime:3D7F:/
+779.2 "Eden's Gravity" sync /:Eden Prime:3D70:/
+790.7 "Eden's Flare" sync /:Eden Prime:3D73:/
+800.2 "Paradise Regained" sync /:Eden Prime:44FC:/
+810.7 "Vice and Virtue (T)" sync /:Eden Prime:3D78:/
+824.2 "Vice and Virtue (DPS)" sync /:Eden Prime:3D7A:/
+836.7 "Enrage" sync /:Eden Prime:45E4:/
```

Comment:
I think technically there can be a delta attack here.  I've seen some folks chat on reddit that depending on if you kill the adds during their enrage or not, you may get all the way to delta attack here.  It's probably worth including the next few mechanics as well, as add phase dps gets faster, seeing more things here is more possible if the final phase is a mess.  I don't know for sure that Spear of Paradise / Heavensunder is next, but it seems like a good guess.

I'd be fine including an "--Enrage Soon--" if you wanted in the 800 range before the jump (or maybe just unrolling the loop)?

Do you know how long the Enrage cast is? If so, could you add a jump to `:45E4:Eden Prime` for the cast start?

Reply:
Interesting; I looked at the longest kills and wipes I could find and I never once saw a Delta Attack, but it _was_ mentioned within the Balance Discord's guide for Eden Prime. I just thought they took an educated guess as to what happened next since I didn't see it in any of the logs I saw.

Enrage is about 8-9 seconds if I remember correctly. I can do a bit more digging into this tomorrow.